.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: node_js
 node_js:
-    - "4"
```

Comment:
Is this a breaking change if we stop supporting 4/6?

Reply:
Yes, in my opinion it is